server/controllers/traveller.js

```diff
@@ -204,11 +205,26 @@ router.post('/addComment', (req, res) => {
   // + '&remoteip=' +
   // req.connection.remoteAddress;
 
-  request(verificationURL, (error, response, body) => {
-    // TODO - new cost
-    // eslint-disable-next-line no-param-reassign
-    body = JSON.parse(body);
-    if (body.success) {
+  const processAddComment = (callback) =>
+    {
+      if (req.body.uid) {
+        checkToken(req, res, req.body.uid, callback);
+      } else {
+       request(verificationURL, (error, response, body) => {
+          // TODO - new cost
```

Comment:
@tom-gregorovic co myslis pod tym TODOckom new cost?

Reply:
To tam bylo i před mou změnou, takže netuším.